setup.sh

```diff
@@ -106,40 +122,48 @@ validate_non_empty() {
 
 # Function to initialize .env file
 initialize_env_file() {
+    local NODE_ENV="development"
     local NETWORK="testnet"
     local MNEMONIC=$(node_modules/.bin/ts-node ./scripts/generate-mnemonic.ts)
     local CORS_ENABLED="false"
     local CORS_ORIGIN="*"
     local NGROK_ENABLED="false"
     local NGROK_AUTHTOKEN=""
     local NGROK_DOMAIN=""
+    local API_URL="http://localhost:4000/api"
 
-    if ask_for_confirmation "Do you want to enable CORS?"; then
-        CORS_ENABLED="true"
-        CORS_ORIGIN=$(read_input_and_validate "Please enter your CORS_ORIGIN (e.g. http://localhost:3000) or press Enter to use the default [*]:" "*" validate_non_empty)
-    else
-        CORS_ENABLED="false"
-        CORS_ORIGIN="*"
+    local TELEGRAM_BOT_TOKEN=$(read_input_and_validate "Enter TELEGRAM_BOT_TOKEN (e.g. 1234567890:ABCdefGHIjklMNoPQRsTUVwxyZ):" "" validate_telegram_bot_token)
```

Comment:
Change the flow a bit to implement it more straightforward for the user. Added a few new parameters for the client side